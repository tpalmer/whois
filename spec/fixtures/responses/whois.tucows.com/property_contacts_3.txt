Registrant:
 Daniel Austin
 3 Gunnery Terrace
 Milverton
 Leamington Spa, Warwickshire CV32 5PE
 UK

 Domain name: SMARTPHRASE.COM


 Administrative Contact:
    Austin, Daniel  123reg@danielaustin.co.uk
    3 Gunnery Terrace
    Milverton
    Leamington Spa, Warwickshire CV32 5PE
    UK
    01926 778794
 Technical Contact:
    Ltd, Webfusion  services@123-reg.co.uk
    5 Roundwood Avenue
    Stockley Park
    Uxbridge, Middlesex UB11 1FF
    GB
    +44.8712309525    Fax: +44.8701650437



 Registration Service Provider:
    123-reg.co.uk, services@123-reg.co.uk
    0845 450 2310
    08709126612 (fax)
    http://www.123-reg.co.uk
    This company may be contacted for domain login/passwords,
    DNS/Nameserver changes, and general domain support questions.


 Registrar of Record: TUCOWS, INC.
 Record last updated on 30-Sep-2011.
 Record expires on 02-Dec-2014.
 Record created on 02-Dec-1999.

 Registrar Domain Name Help Center:
    http://tucowsdomains.com

 Domain servers in listed order:
    NS5.ESOSOFT.NET   
    NS6.ESOSOFT.NET   


 Domain status: clientDeleteProhibited
                clientTransferProhibited
                clientUpdateProhibited

The Data in the Tucows Registrar WHOIS database is provided to you by Tucows
for information purposes only, and may be used to assist you in obtaining
information about or related to a domain name's registration record.

Tucows makes this information available "as is," and does not guarantee its
accuracy.

By submitting a WHOIS query, you agree that you will use this data only for
lawful purposes and that, under no circumstances will you use this data to:
a) allow, enable, or otherwise support the transmission by e-mail,
telephone, or facsimile of mass, unsolicited, commercial advertising or
solicitations to entities other than the data recipient's own existing
customers; or (b) enable high volume, automated, electronic processes that
send queries or data to the systems of any Registry Operator or
ICANN-Accredited registrar, except as reasonably necessary to register
domain names or modify existing registrations.

The compilation, repackaging, dissemination or other use of this Data is
expressly prohibited without the prior written consent of Tucows.

Tucows reserves the right to terminate your access to the Tucows WHOIS
database in its sole discretion, including without limitation, for excessive
querying of the WHOIS database or for failure to otherwise abide by this
policy.

Tucows reserves the right to modify these terms at any time.

By submitting this query, you agree to abide by these terms.

NOTE: THE WHOIS DATABASE IS A CONTACT DATABASE ONLY.  LACK OF A DOMAIN
RECORD DOES NOT SIGNIFY DOMAIN AVAILABILITY.


