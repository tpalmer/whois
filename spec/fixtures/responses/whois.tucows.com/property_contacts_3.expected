#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].name         == "Daniel Austin"
  should: %s[0].organization == nil
  should: %s[0].address      == "3 Gunnery Terrace\nMilverton"
  should: %s[0].city         == "Leamington Spa"
  should: %s[0].zip          == "CV32 5PE"
  should: %s[0].state        == "Warwickshire"
  should: %s[0].country_code == "UK"
  should: %s[0].phone        == nil
  should: %s[0].fax          == nil
  should: %s[0].email        == nil
